libmisc/limits.c

```diff
@@ -29,8 +29,6 @@
 #include "getdef.h"
 #include "shadowlog.h"
 #include <sys/resource.h>
-#define LIMITS
-#ifdef LIMITS
 #ifndef LIMITS_FILE
```

Comment:
I think we can get rid of `#ifndef LIMITS_FILE` and assume that we need to define `LIMITS_FILE` always.

Reply:
Sounds good to me